### DISCUSSION TITLE
Counsil discussions?

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#17] [Q&A]
Examples (non exhaustive) of subjects discussed by the council would be useful

**Comment by ebullient** [comment]
Bootstrap phase: nail down bylaws and the missing details from all of these drafts. Brainstorm on outreach strategies for grassroots growth of foundation members and projects: a necessary element for this foundation to be viable.

The primary on-going concern that I see is allocation of incoming resources, and defining and building out support that member projects need to be successful. 

I don't have an exhaustive list of the possibilities. There are some that I can see, but I don't want to over-anticipate.

**Comment by kenfinnigan** [comment]
What's discussed at meetings is covered at a high level here: https://github.com/commonhaus/foundation-draft/blob/main/bylaws/3-cf-council.md#meetings-procedures-and-decision-making
